patches/components-security_interstitials-content-cert_report_helper.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/components/security_interstitials/content/cert_report_helper.cc b/components/security_interstitials/content/cert_report_helper.cc
+index 448759d7a69eaba26b1daa5a37a5c444bf704d1c..b51348fe4b2a7d5838b488d2015a9a5a6c202375 100644
+--- a/components/security_interstitials/content/cert_report_helper.cc
++++ b/components/security_interstitials/content/cert_report_helper.cc
+@@ -205,6 +205,7 @@ bool CertReportHelper::ShouldShowEnhancedProtectionMessage() {
+   // the window is not incognito AND
+   // Safe Browsing is not managed by policy AND
+   // the user is not already in enhanced protection mode.
++  BRAVE_CERT_REPORT_HELPER_SHOULD_SHOW_ENHANCED_PROTECTION_MESSAGE
```

Comment:
isn't this already covered by `IsEnhancedProtectionEnabled`?
```
bool is_enhanced_protection_enabled =
      safe_browsing::IsEnhancedProtectionEnabled(*pref_service);
  bool is_safe_browsing_managed =
      safe_browsing::IsSafeBrowsingPolicyManaged(*pref_service);

  if (in_incognito) {
    return false;
  }
  if (is_enhanced_protection_enabled) {
    return false;
  }
```

Reply:
This one, sadly, checks the opposite. So if the `is_enhanced_protection_enabled` is `false`, like in our case, `ShouldShowEnhancedProtectionMessage` will return `true` and will show the safe browsing text on the interstitial page.

Reply:
I see, so I don't see a way around this patch then, but I think we should be able to get rid of https://github.com/brave/brave-core/pull/8307/commits/654388e9514315fda64609a092d386ef52033420#diff-863da2c54f3691cd10286cd97f82a69475dba820193aedfaddc6986f05d516e4R9